### ISSUE TITLE
Get Support screen instead of Network error displays when client app has the connection problem

### ISSUE THREAD

**Issue by longbmk** [bitmark-inc/autonomy-apps#2359]
Hi Anais, the issue is about the network connection, We have an error code. I think in this case, the UX should not show to “Get Support”. It should be a message to notify the user re-connect because we can’t help in this issue. The report function cannot help it better. Do you agree to it Anais? If yes, can you please provide the screen for it. We will show this message when we got this error code. 

Can you please help to have a UI in here? Thank you.

https://bitmark.slack.com/archives/C03FX9ZC90B/p1675806387702939.

https://github.com/bitmark-inc/au-support-board/issues/1117

**Comment by Anais-bitmark** [comment]
@longbmk 

Here the design for this ticket

[Figma](https://www.figma.com/file/0JRIvF8ACy8PcPGsuJusX9/bug_2359%3A-Get-Support-screen-instead-of-Network-error-displays-when-client-app-has-...?node-id=0%3A1&t=fSglqxbD9UTygUuU-1)

<img width="524" alt="Capture d’écran 2023-02-20 à 11 29 38" src="https://user-images.githubusercontent.com/61187455/220094285-826583b2-41a7-48d6-86cc-3740812134ca.png">

**Comment by phuocbitmark** [comment]
@Anais-bitmark actually, in this case, we(the app) don't know what kind of error it is. we can determine only after reading the debug log. 
as @longbmk said, if we put "get support" there, we will be overflow with "unvaluable" debug logs.
I'm not sure what would be appropriate to put there.
Do you have any idea

**Comment by Anais-bitmark** [comment]
@phuocbitmark 
I'm just seeing your answer now.
So when the error occur we are not able to determine the reason right?
What the user should do?
Relaunch the app ?

**Comment by phuocbitmark** [comment]
yes, since we don't know what is the reason, we can first admit that we don't know, then suggest them to do: check the internet condition, relaunch and do it again, (please add more if you can think of any). and finally if all of these above does not work, tell user to go to Help and send a debug log.
these logs are usually not so valuable, so to save effort on research and reply on customer support, don't show button to attach debug log, they may tap it without doing things that we suggested.
